graphics/vtk/Portfile

```diff
@@ -14,15 +14,17 @@ compiler.cxx_standard 2011
 compiler.blacklist-append {clang < 900}
 
 name                vtk
-version             9.1.0
-revision            4
+version             9.2.6
+revision            0
 categories          graphics devel
 platforms           darwin
 license             BSD
 
 set branch          [join [lrange [split ${version} .] 0 1] .]
 
-maintainers         {stromnov @stromnov} {@rdbisme gmail.com:rubendibattista} openmaintainer
+maintainers         {stromnov @stromnov} \
+                    @Dave-Allured \
```

Comment:
Generally, it's expected that you'll provide a (obfuscated) email address so you can be contacted if there's trouble with your port. See the maintainer entry you are removing.